storage/client/client.go

```diff
@@ -1504,15 +1504,24 @@ func (c *StorageClient) RuntimeAccount(ctx context.Context, address staking.Addr
 		&totalSent,
 		&totalReceived,
 		&a.Stats.NumTxns,
-	); err != nil {
-		return nil, wrapError(err)
-	}
-	a.Stats.TotalSent, err = common.NumericToBigInt(totalSent)
-	if err != nil {
-		return nil, wrapError(err)
-	}
-	a.Stats.TotalReceived, err = common.NumericToBigInt(totalReceived)
-	if err != nil {
+	)
+
+	switch err {
+	case nil:
+		a.Stats.TotalSent, err = common.NumericToBigInt(totalSent)
+		if err != nil {
+			return nil, wrapError(err)
+		}
+		a.Stats.TotalReceived, err = common.NumericToBigInt(totalReceived)
+		if err != nil {
+			return nil, wrapError(err)
+		}
+	case pgx.ErrNoRows:
+		// If an account address has no activity, default to 0.
+		a.Stats.TotalSent = common.NewBigInt(0)
+		a.Stats.TotalReceived = common.NewBigInt(0)
+		a.Stats.NumTxns = 0
```

Comment:
Your call on whether it's actually better, but you _could_ apply defaults in SQL:
```sql
SELECT
  COALESCE(computed.total_sent, 0),
  COALESCE(computed.total_received, 0),
  COALESCE(computed.num_txs, 0)
FROM
(VALUES(NULL)) AS dummy -- even if SELECT below matches no accounts, ensures that there will be a row to return
LEFT JOIN (
	SELECT total_sent, total_received, num_txs
	FROM chain.runtime_accounts
	WHERE
		(runtime = $1) AND
		(address = $2::text)
) AS computed ON true;
```

Reply:
Hm I'd prefer to do it here and keep all the logic in one file.